farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html

```diff
@@ -100,21 +141,82 @@ <h1 style='text-align:center;'>Seeding Input Log</h1>
+                    this.numSeedsPlanted = newNumSeedsPlanted
+                },
+                setNewNumRowBed(newNumRowBed) {
+                    this.numRowBed = newNumRowBed
+                },
+                setNewNumFeet(newNumFeet) {
+                    this.numFeet = newNumFeet
+                },
+                setNewNumWorkers(newNumWorkers) {
+                    this.numWorkers = newNumWorkers
+                },
+                setTime(newTime) {
+                    this.time = newTime
+                },
+                clearFields(){
+                    this.selectedDate = dayjs().format('YYYY-MM-DD').toString() 
+                    this.selectedCrop = null 
+                    this.selectedArea = null
+                    this.savedArea = null
+                    this.selectedSeedingType = ''
+                    this.selectedFeetUnit = 'Bed Feet'
+                    this.selectedTimeUnit = 'minutes'
+                    this.comments = ''
+                    this.numWorkers = null
+                    this.time = null
+                    this.numOfTrays= null
+                    this.numCellsPerTray= null
+                    this.numSeedsPlanted= null
+                    this.numRowBed= null
+                    this.numFeet= null
+                },
+                submitLog() {
+                    this.isSubmitted = true
+                },
+                saveLog() {
                     createRecord('/farm_asset', this.plantingLogData, this.sessionToken).then((response) => {
                         this.plantingId = response.data.id
                     }).then(() => {
                         createRecord('/log', this.logData, this.sessionToken).then((response) => {
-                        })
+                            this.clearFields();
+                        })    
                     })
-                    if(confirm("You have saved a log! Would you like to go the seeding report page?")){
-                        window.location.replace('/farm/fd2-barn-kit/seedingReport')
-                    }
+                    this.alertType = 'success'
+                    this.isSubmitted = false
+                    setTimeout(() => {
+                        this.alertType = '' 
+                    }, 3000)    
+                },
+                cancelLog() {
+                    this.isSubmitted = false
+                    this.alertType = 'canceled'
+                    setTimeout(() => { 
+                        this.alertType = ''
+                    }, 3000)
                 },
+                regTimeSwitch(){
+                    if (this.selectedTimeUnit == "minutes") {
+                        this.regTime = "^[1-9]+[0-9]*$"
+                        this.time = null
+                    } else if (this.selectedTimeUnit == "hours") {
+                        this.regTime = "^[1-9]+[0-9]*(\.[0-9]{1,2}){0,1}$|^[0]{0,1}\.[1-9][0-9]{0,1}$|^[0]{0,1}[1-9]$|^[1-9]+[0-9]*\.$"
```

Comment:
save this into a diff data  var and get rid of this func as we are using v-if (v-show)